### PR TITLE
add selectable levelhead mode

### PR DESCRIPTION
**Pull request by moehreag**
Created in collaboration with @Lolonaut679 (he wants to learn java)
image

Modes until now: 
	- Network Level
	- Bedwars Level
	- Skywars Level

### CODE REVIEW

File: src/main/java/com/intro/client/util/HypixelAbstractionLayer.java [hunk 1]
```diff
@@ -49,7 +50,17 @@ public static boolean canUseHypixelService() {
     public static int getPlayerLevel(String uuid) {
        if(loadPlayerDataIfAbsent(uuid)) {
            try {
-               return (int) cachedPlayerData.get(uuid).get(1, TimeUnit.MICROSECONDS).getPlayer().getNetworkLevel();
+
+               Enum<?> mode = OsmiumClient.options.getEnumOption(Options.LevelHeadMode).get();
+               System.out.println(cachedPlayerData.get(uuid).get(1, TimeUnit.MICROSECONDS).getPlayer().getRaw());
```
Comment (Intro-Dev) [inline src/main/java/com/intro/client/util/HypixelAbstractionLayer.java]: This prints gotta be removed

**Review by Intro-Dev — CHANGES_REQUESTED** [review]
Looks good, other than the println in there

File: src/main/java/com/intro/client/util/HypixelAbstractionLayer.java (hunk 1 above)
Comment (moehreag) [reply src/main/java/com/intro/client/util/HypixelAbstractionLayer.java]: I really don't know how that slipped through..

**Review by Intro-Dev — APPROVED** [review]
Looks good

**Pull request merged**